client/src/components.d.ts

```diff
@@ -7,8 +7,10 @@ export {}
 /* prettier-ignore */
 declare module 'vue' {
   export interface GlobalComponents {
-    AppFooter: typeof import('./components/AppFooter.vue')['default']
-    HelloWorld: typeof import('./components/HelloWorld.vue')['default']
+    AlreadyHaveAnAccount: typeof import('./components/AlreadyHaveAnAccount.vue')['default']
+    LoginGithub: typeof import('./components/LoginGithub.vue')['default']
+    LoginTFT: typeof import('./components/LoginTFT.vue')['default']
```

Comment:
```suggestion
    TFLogin: typeof import('./components/TFLogin.vue')['default']
```

Why do we use `typeof` and `['default']`?